controllers/vrg_volrep.go

```diff
@@ -1889,7 +1892,13 @@ func (v *VRGInstance) restorePVsAndPVCsFromS3(result *ctrl.Result) (int, error)
 		// Ignoring PVC restore errors helps with the upgrade from ODF-4.12.x to 4.13
 		pvcCount, err = v.restorePVCsFromObjectStore(objectStore, s3ProfileName)
 
-		if err != nil || pvCount != pvcCount {
+		if err != nil {
+			v.log.Error(err, fmt.Sprintf("error fetching PVC cluster data from S3 profile %s", s3ProfileName))
```

Comment:
Same comment about the message and log level.

Reply:
Changed to warning, added new context